packages/features/portfolio/src/containers/portfolio-risk/components/vault-tab.tsx

```diff
@@ -0,0 +1,124 @@
+import { observer } from 'mobx-react-lite';
+import { Box, styled } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { FormattedMessage } from 'react-intl';
+
+import { TokenIcon } from '@notional-finance/icons';
+import { useSelectedNetwork } from '@notional-finance/notionable-hooks';
+import { Body, Button, Caption, H4, LabelValue } from '@notional-finance/mui';
+import { useVaultRiskTable } from '@notional-finance/portfolio-feature-shell/hooks';
+
+const VaultTab = () => {
+  const theme = useTheme();
+  const network = useSelectedNetwork();
+  const { riskTableData } = useVaultRiskTable();
+
+  return (
+    <Container>
+      <Box>
+        {riskTableData.map((item) => (
+          <Card>
+            <TokenContainer>
+              <TokenIcon
+                symbol={item.vault.symbol}
+                size={'xl'}
+                style={{ width: 40, height: 40 }}
```

Comment:
Similar comment to the other one, shouldn't the `size` prop already handle the width / height